CWE-94-Python_darwin/KasRoudra_k-fuscator/kf.py

```
@@ -0,0 +1,273 @@
+    if ret=="1":
+        main()
+    else: 
+        exit()
+
+# Custom path chooser
+def mover(out_file):
+    move= input(ask+"Move to a custom path?(y/n) > "+green)
+    if move=="y":
+        mpath=input(ask+"Enter the path > "+ green)
+        if os.path.exists(mpath):
+            os.system(f'''mv -f "{out_file}" "{mpath}" ''')
```

Ruff (0.13.2)

Starting a process with a shell, possible injection detected

(S605)